setup.py

```diff
@@ -3,6 +3,7 @@
 from setuptools import setup
 
 from pathlib import Path
+from replicate.package_version import __version__
```

Comment:
This doesn't work unforutnately because this script needs to execute to read the dependencies to be able to import `replicate`. Python packaging lol

Pick your poison https://packaging.python.org/en/latest/guides/single-sourcing-package-version/

[I am partial to (3).](https://github.com/docker/docker-py/blob/master/setup.py#L37)

Reply:
Ah I missed your comment. Isn't it hilarious how those hacks are all officially documented. Everything about Python packaging is offensive so it doesn't really matter what we do.

Reply:
Yay! That worked. This is ready for another look.